INSTALL.md

```diff
@@ -53,14 +60,23 @@ docker exec -it [container_id] bash
 # Move to the application root dir
 cd src
 
+# Run development server
+python manage.py runserver 0.0.0.0:8080
+
+# Access to the development server
+http://0.0.0.0:8080
+
+# Access to Nginx server
+http://127.0.0.1
+
 # Run existing migrations
-python3 manage.py migrate
+python manage.py migrate
```

Comment:
The application use python3 and in the docker container python and python3 is the same, it looks redundant for me the python3.